src/lib/main/core/cell/cell.service.ts

```diff
@@ -71,33 +71,23 @@ export class CellService {
 		if (!canBuild(column)) {
 			return noop;
 		}
-
-		const id = buildId(source, column, mode);
-		const commits = this.commits;
-		let commit = commits.get(column.key);
+		let commit = this.commits.get(column.key);
 		if (commit) {
 			return commit;
 		}
-
-		const templateService = this.templateService;
 		const keys = buildKeys(source, column, mode);
-		const link = templateService.find(keys);
-
-		if (!link) {
-			throw new AppError(
-				'cell.service',
-				`Can't find template for ${keys[0]}`
-			);
+		const templateLink = this.templateService.find(keys);
+		if (!templateLink) {
+			console.warn(`Can't find template for ${keys[0]}`);
+			return noop;
 		}
-
 		commit = (container: ViewContainerRef, context: any) => {
 			container.clear();
-
-			const createView = templateService.viewFactory(context);
-			createView(link, container);
+			const createView = this.templateService.viewFactory(context);
+			createView(templateLink, container);
 		};
-
-		commits.set(id, commit);
+		const id = buildId(source, column, mode);
```

Comment:
hmm we build id in one way and then get commit by column.key, is it not strange?